Kconfig

```diff
@@ -402,6 +402,11 @@ menu "LVGL configuration"
 					linear: 4K bytes.
 					radial: radius * 4K bytes.
 
+		config LV_VG_LITE_STROKE_CACHE_CNT
```

Comment:
If you know the expected memory usage, please add it in the description.

Reply:
Memory usage is unpredictable because it is strongly related to the length of the path data, and there is no length limit for the path data.